include/bluetooth/hci_vs_sdc.h

```diff
@@ -392,6 +392,28 @@ int hci_vs_sdc_scan_accept_ext_adv_packets_set(
  */
 int hci_vs_sdc_set_role_priority(const sdc_hci_cmd_vs_set_role_priority_t *params);
 
+/** @brief Set Event Start Task.
+ *
+ * For the complete API description, see sdc_hci_cmd_vs_set_event_start_task().
```

Comment:
```suggestion
 * For the complete API description, see @ref sdc_hci_cmd_vs_set_event_start_task().
```
Will that work? would be nicer to have clickable link.
Actually I was not able to find generated docs for it here https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16549/nrf/index.html. Do you know where is it placed?

Reply:
Refs from sdk-nrf to nrfxlib do not work AFAIK. I've asked a question about it here: https://github.com/nrfconnect/sdk-nrf/pull/16550#discussion_r1687766121.
@greg probably knows this better